### DISCUSSION TITLE
Output when validating is not generating.  Please help.

### DISCUSSION THREAD

**Discussion by winnkathy-hub** [CMSgov/price-transparency-guide-validator#118] [Q&A]
when trying to validate  any of the schemas, the validator is not returning any OUTPUT (not even to a file).   The only line I get is  "Running validatoar container ..."    for every file with no output.    I even tried to send the output to a text file and it doesn't work.

The code below is the example on validating a bad file.  So I know I should see some type of error output.

cms-mrf-validator validate test-files/allowed-amounts-error.json --schema-version v1.0.0 -o "C:\Users\regwinka2\results.txt" 
-t allowed-amounts

Can someone please help me with this issue?
Thank You!

**Comment by shaselton-usds** [comment]
@winnkathy-hub Would it be possible to provide a bit more information? What version of docker are you using, do you have the latest validator code? Have you been able to run the validator successfully in the past? Did the output change when upgrading? etc...

Thanks!
